index.test.js

```diff
@@ -0,0 +1,87 @@
+const {
+  map,
+  filter,
+  findIndex,
+  every,
+  reduce
+} = require('./index');
+
+describe('map', () => {
+  it('takes an array and a callback and creates a new array with the return value of each called callback.', () => {
+    const callback = (item) => item.toUpperCase();
+    const arr = ['bob', 'ben'];
+    const expected = ['BOB', 'BEN'];
+    const actual = map(arr, callback);
+
+    expect(actual).toEqual(expected);
+    expect(arr).toEqual(arr);
+  });
+});
+
+describe('filter', () => {
+  it('takes an array and a callback and returns a new array with all items whose callback returned true.', () => {
+    const callback = (item) => {
+      if(item.startsWith('b') === true)
+        return item;
+    };
+    const arr = ['bob', 'ben'];
```

Comment:
this is picky, but as feedback, might want to see another test that has something that doesn't begin with 'b'.  That way you know it filters out, for example the name 'angela'. That being said though, can tell this works as intended. Nice job, these look great!